spring-integration-cassandra/src/test/java/org/springframework/integration/cassandra/config/CassandraParserUtilsTest.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.integration.cassandra.config;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.springframework.beans.factory.config.BeanDefinition;
+import org.springframework.beans.factory.support.RootBeanDefinition;
+import org.springframework.integration.cassandra.config.xml.CassandraParserUtils;
+
+/**
+ * @author Filippo Balicchia
+ */
+
+public class CassandraParserUtilsTest {
```

Comment:
Not sure if we need to test this stuff, but it won't hurt.
Anyway the test classes should be named like `*Tests`, not in singular.
Will rename on merge.